config/inlineInheritDoc.ts

```diff
@@ -134,45 +136,48 @@ function loadApiModel() {
   }
 }
 
-function processComments(model: ApiModel) {
+function processComments(model: ApiModel, options: BuildStepOptions) {
   const inheritDocRegex = /\{@inheritDoc ([^}]+)\}/;
 
-  const project = new Project({
-    tsConfigFilePath: "tsconfig.json",
-    skipAddingFilesFromTsConfig: true,
+  return applyRecast({
+    glob: `**/*.{${options.jsExt},d.${options.tsExt}}`,
+    cwd: options.targetDir,
+    transformStep({ ast, sourceName }) {
+      return {
+        ast: visit(ast, {
+          visitNode(path) {
+            const node = path.node;
+            if (node.comments)
+              for (const comment of node.comments) {
+                if (comment.type === "CommentBlock") {
+                  let newText = comment.value;
+                  while (inheritDocRegex.test(newText)) {
+                    newText = newText.replace(
+                      inheritDocRegex,
+                      (_, canonicalReference) => {
+                        return getCommentFor(canonicalReference, model) || "";
+                      }
+                    );
+                  }
+                  if (comment.value !== newText) {
+                    comment.value = frameComment(newText);
+                  }
+                }
+              }
+            return this.traverse(path);
```

Comment:
```suggestion
            if (node.comments) {
              for (const comment of node.comments) {
                if (comment.type === "CommentBlock") {
                  let newText = comment.value;
                  while (inheritDocRegex.test(newText)) {
                    newText = newText.replace(
                      inheritDocRegex,
                      (_, canonicalReference) => {
                        return getCommentFor(canonicalReference, model) || "";
                      }
                    );
                  }
                  if (comment.value !== newText) {
                    comment.value = frameComment(newText);
                  }
                }
              }
            }
            return this.traverse(path);
```

Since there gets to be quite a bit of indentation inside the `if`, can we wrap with brackets so its easier to tell where it ends?

Reply:
Good point. I'll do one better and add an early return.

Reply:
44cb005af989500f9e2a598070635dfd72cac6fc